bin/oneoff/data_fix/missing_reference_guides.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+# https://codedotorg.atlassian.net/browse/TEACH-840
+# CSD-2023 and CSD-2024 don't have any reference guides linked to them
+# because of an issue in the way these units were originally cloned.
+# This script clones the reference guides which were missed and links
+#  them to the appropriate courses.
+
+require_relative '../../../dashboard/config/environment'
+
+def data_fix_reference_guides
+  raise unless Rails.application.config.levelbuilder_mode
+
+  # List of course versions to be fixed
+  course_version_to_fix = CurriculumHelper.find_matching_course_version('csd-2024')
```

Comment:
Do you also need to look up and fix `'csd-2023'` here, or does that somehow happen for free when you fix the 2024 version?